ogr/ogrgeometry.cpp

```diff
@@ -4703,6 +4711,101 @@ OGRGeometryH OGR_G_UnionCascaded(OGRGeometryH hThis)
         OGRGeometry::FromHandle(hThis)->UnionCascaded());
 }
 
+/************************************************************************/
+/*                               UnaryUnion()                           */
+/************************************************************************/
+
+/**
+ * \brief Returns the union of all components of a single geometry.
```

Comment:
I do not quite understand what this comment means.

Reply:
I've just stolen it from upstream GEOS function...  : https://libgeos.org/doxygen/3.11/geos__c_8h.html#ada5b7b39a81b722b97371736ff0b8d29

Perhaps adding a link to https://postgis.net/docs/ST_UnaryUnion.html if it makes things clearer ?

Reply:
done